src/main/java/com/plaid/client/request/InstitutionsGetByIdRequest.java

```diff
@@ -25,15 +26,14 @@ public InstitutionsGetByIdRequest(String institutionId) {
   }
 
   public InstitutionsGetByIdRequest withIncludeOptionalMetadata(boolean includeOptionalMetadata) {
-    this.options = new Options(includeOptionalMetadata);
```

Comment:
wanna include "include_status" with this too?

Reply:
Michael was planning on handle that

Reply:
⚡️